### PR TITLE
Add Workspace Selection guidance

### PR DESCRIPTION
**Pull request by csarven**

### CODE REVIEW

**Review by VirginiaBalseiro — APPROVED** [review]

**Review by michielbdejong — APPROVED** [review]

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -200,7 +200,26 @@ Congratulations :tada: Solid community thanks you :sparkles:.
 ## Work items
 
 In general, all documents in scope of the group are welcome. Work items can be
-documents or software.
+documents or software - typically Technical Reports, Primers, Best Practices
+and Guidelines, Use Cases and Requirements, Surveys, QA, and so forth.
```
Comment (TallTed) [inline CONTRIBUTING.md:204]:
```suggestion
documents — such as Technical Reports, Primers, Best Practices and Guidelines,
Use Cases and Requirements, Surveys, QA, and so forth — or software.
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -200,7 +200,26 @@ Congratulations :tada: Solid community thanks you :sparkles:.
...
+
+### Workspace Selection
+
+Consider the following guidance to identify the appropriate workspace to
+efficiently manage communication and collaboration within the Solid Project.
```
Comment (TallTed) [inline CONTRIBUTING.md:209]:
```suggestion
Consider the following guidance to identify the appropriate workspace to use to
efficiently manage communication and collaboration within the Solid Project.
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -200,7 +200,26 @@ Congratulations :tada: Solid community thanks you :sparkles:.
...
+
+When the work in consideration:
+
+* involves or spans across various stakeholders, specialized groups, or
+  horizontal grouping, then start a Solid Project-wide endeavor.
+* is about current CG work items, then it should be proposed within or across
+  relevant work.
+* is within the scope of the CG but identifies a gap in work items, then it
+  can follow the procedure to [proposing a New Work
+  Item](#new-work-item-proposal)
```
Comment (TallTed) [inline CONTRIBUTING.md:219]:
```suggestion
* involves or spans over various stakeholders, specialized groups, or
  horizontal groupings, then start a Solid Project-wide endeavor
* is about current CG work items, then it should be proposed within or across
  those current work items
* is within the scope of the CG but identifies a gap in work items, then
  follow the procedure to [propose a New Work Item](#new-work-item-proposal)
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -200,7 +200,26 @@ Congratulations :tada: Solid community thanks you :sparkles:.
...
+
+If you are unsure where the work may fit, please create an issue detailing the
+stakeholders, problem context, and technical or knowledge goals.
```
Comment (TallTed) [inline CONTRIBUTING.md:222]: Should indicate on which repo to create that issue, preferably including a link

**Review by TallTed — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -200,7 +200,25 @@ Congratulations :tada: Solid community thanks you :sparkles:.
...
+
+* involves or spans over various stakeholders, specialized groups, or
+  horizontal groupings, then start a Solid Project-wide endeavor
+* is about current CG work items, then it should be proposed within or across
+  those current work items
+* is within the scope of the CG but identifies a gap in work items, then
+  follow the procedure to [propose a New Work Item](#new-work-item-proposal)
+
+If you are unsure where the work may fit, please create an issue detailing the
+stakeholders, problem context, and technical or knowledge goals.
```
Comment (csarven) [inline CONTRIBUTING.md:221]:
```suggestion
If you are unsure where the work may fit, please create an
[issue](https://github.com/solid/specification) detailing the stakeholders,
problem context, and technical or knowledge goals.
```

**Review by elf-pavlik — APPROVED** [review]

**Pull request merged**